### ISSUE TITLE
material-yes-button-text-color mixin don't work with stepper component

### ISSUE THREAD

**Issue by muzuro** [dart-lang/angular_components#279]
I created component with template:
```
<div class="teal-themed">
    <material-stepper legalJumps="backwards"
                      orientation="horizontal"
                      size="default">
        <template step name="Consent">
            <div>
            </div>
        </template>
        <template step name="Strips">
            <div>
            </div>
        </template>
    </material-stepper>
</div>
```
and scss: 
```
@import 'package:angular_components/css/material/material';
@import 'package:angular_components/material_stepper/mixins';
@import 'package:angular_components/material_yes_no_buttons/mixins';

:host {
  @include material-stepper-theme (
          $selector: '.teal-themed',
          $step-color: $mat-teal-500,
          $button-color: $mat-blue-400
  );
  .teal-themed {
    @include material-no-button-text-color($mat-teal-500);
    @include material-yes-button-text-color($mat-teal-500);
  }
}
```
Strange that this code override color for `no` button but is ignored for `yes`. Styles from asset:angular_components/lib/material_yes_no_buttons/material_yes_no_buttons.dart is not override by mixin - as I see it in chrome dev tools.

**Comment by TedSander** [comment]
Thanks for the bug. The yes button is raised and highlighted which is causing it to not have a great enough specificity on the CSS selector. I'll try to get a fix in the next sync.

**Comment by nshahan** [comment]
Should be resolved in v0.9.0-beta+1. Feel free to reopen if needed.
